src/test/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigatorTraitsTest.java

```diff
@@ -364,1200 +273,775 @@ public void limit_branches() throws Exception {
-                                hasProperty("strategyId", is(2))
-                        ),
-                        Matchers.allOf(
-                                instanceOf(ForkPullRequestDiscoveryTrait.class),
-                                hasProperty("strategyId", is(1))
-                        )
-                )
-        );
+                        Matchers.allOf(instanceOf(OriginPullRequestDiscoveryTrait.class),
+                                hasProperty("strategyId", is(2))),
+                        Matchers.allOf(instanceOf(ForkPullRequestDiscoveryTrait.class),
+                                hasProperty("strategyId", is(1)))));
     }
 
     @Test
     public void build_000111() throws Exception {
         GitHubSCMNavigator instance = load();
         assertThat(instance.getTraits(),
                 containsInAnyOrder(
-                        Matchers.allOf(
-                                instanceOf(OriginPullRequestDiscoveryTrait.class),
-                                hasProperty("strategyId", is(2))
-                        ),
-                        Matchers.allOf(
-                                instanceOf(ForkPullRequestDiscoveryTrait.class),
-                                hasProperty("strategyId", is(3))
-                        )
-                )
-        );
+                        Matchers.allOf(instanceOf(OriginPullRequestDiscoveryTrait.class),
+                                hasProperty("strategyId", is(2))),
+                        Matchers.allOf(instanceOf(ForkPullRequestDiscoveryTrait.class),
+                                hasProperty("strategyId", is(3)))));
     }
 
     @Test
     public void build_001000() throws Exception {
         GitHubSCMNavigator instance = load();
         assertThat(instance.getTraits(),
-                contains(
-                        Matchers.allOf(
-                                instanceOf(OriginPullRequestDiscoveryTrait.class),
-                                hasProperty("strategyId", is(1))
-                        )
-                )
-        );
+                contains(Matchers.allOf(instanceOf(OriginPullRequestDiscoveryTrait.class),
+                        hasProperty("strategyId", is(1)))));
     }
 
     @Test
     public void build_001001() throws Exception {
         GitHubSCMNavigator instance = load();
         assertThat(instance.getTraits(),
                 containsInAnyOrder(
-                        Matchers.allOf(
-                                instanceOf(OriginPullRequestDiscoveryTrait.class),
-                                hasProperty("strategyId", is(1))
-                        ),
-                        Matchers.allOf(
-                                instanceOf(ForkPullRequestDiscoveryTrait.class),
-                                hasProperty("strategyId", is(2))
-                        )
-                )
-        );
+                        Matchers.allOf(instanceOf(OriginPullRequestDiscoveryTrait.class),
```

Comment:
Here the auto formatting has made it less readable than before.

Reply:
Agree, we should get used to it, I guess